src/DependencyInjection/Configuration.php

```diff
@@ -51,42 +51,26 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->end()
                 ->end()
                 ->scalarNode('auto_generate_proxy_classes')
+                //->integerNode('auto_generate_proxy_classes')
```

Comment:
By using `ifTrue` and `ifFalse`, boolean should be allowed.
Error with the `intergetNode`:

```
Invalid type for path "doctrine_mongodb.auto_generate_proxy_classes". Expected "int", but got "bool".
src/Symfony/Component/Config/Definition/IntegerNode.php:26
```